src/raft.rs

```diff
@@ -236,6 +236,13 @@ pub struct RaftCore<T: Storage> {
     /// Enable this if greater cluster stability is preferred over faster elections.
     pub pre_vote: bool,
 
+    /// Enable follower replication.
+    ///
+    /// This enables data replication from a follower to other servers in the same available zone.
+    ///
+    /// Enable this for reducing across-AZ traffic of cloud deployment.
+    pub follower_repl: bool,
```

Comment:
Why `pub`?

Reply:
`raftstore` will check whether follower replication is enabled in order to merge `MsgAppend` in `fn build_raft_messages`.

Reply:
Resolved.